### ISSUE TITLE
withCredentials can be provided directly in options

### ISSUE THREAD

**Issue by naugtur** [AmpersandJS/ampersand-sync#47]
`xhr` supports `withCredentials` option for a while now and the docs describing aompersand-sync should not say that you need `xhrFields` to add `withCredentials` as a typical case.

(help me find the file and I can PR it)

**Comment by pgilad** [comment]
https://github.com/AmpersandJS/ampersand-model/blob/master/README.md#ajaxconfig-modelajaxconfig-or-modelajaxconfig

**Comment by naugtur** [comment]
There's been much more changes since I created this issue... 
Thanks for pointing to those.
I see other modules are documented in READMEs so it's probably best to doscument sync in its README first, and then fill in the above.

**Comment by pgilad** [comment]
Yeah I agree.. Although a lot of the `ajaxConfig` are similar between any module that relies on `&-sync`
